subgraph/.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+const path = require("path");
+
+module.exports = {
+  root: true,
+  extends: ["../.eslintrc.js", "plugin:@typescript-eslint/recommended"],
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint"],
```

Comment:
Why did you add `@typescript-eslint` here? This one is added for root eslint conifg.

Reply:
It has been removed!